Dev app codetest

sipe-daniel opened the pull request.

sipe-daniel committed:
code overview

sipe-daniel committed:
:recycle:refactor(customTextField.dart, joinRoom.dart) : add customtextfield code and refactor joinRoom code

sipe-daniel committed:
Merge branch 'dev-app' of https://github.com/Un-dev/Netlab into dev-app

tchawou-daniel committed:
:construction: feat add poll section

tchawou-daniel committed:
refactor poll page

Un-dev commented:
@sipe-daniel dis moi quand tu as fini pour que je puisse merge les conflits

tchawou-daniel committed:
:construction: add the section films description

Un-dev commented on netlab/lib/Screens/Home/home.dart:
```diff
@@ -25,6 +25,12 @@ class Home extends StatelessWidget {
                   Navigator.of(context).pushNamed('/joinroom');
                 },
               ),
+              RaisedButton(
```
Utilise le widget CustomButton ici, si tu regarde le [home.dart](https://github.com/Un-dev/Netlab/blob/dev-app/netlab/lib/Screens/Home/home.dart) de la branch dev-app j'ai utilisé CustomButton partout
```suggestion
              CustomButton(
```

sipe-daniel replied:
J'ai remplacé RaisedButton par CustomButtom ICI

Un-dev commented on netlab/lib/Screens/Room/poll.dart:
```diff
@@ -0,0 +1,127 @@
+                      Container(
+                        padding: const EdgeInsets.fromLTRB(100.0, 0.0, 0.0, 10.0),
+                        child: Row(
+                            children: <Widget>[
+                              Text('Note générale ', style: TextStyle(color: Colors.white,fontSize: 15), textAlign: TextAlign.center),
+                              for (int i = 0; i < 3; i++)
+                                Icon(MyIcons.star,color: Colors.yellow,size: 20,),
+                              for (int i = 0; i < 2; i++)
+                                Icon(MyIcons.star_border,color: Colors.yellow,size: 20,),
+                            ]
+                        ),
+                      ),
+                      Container(
+                        padding: const EdgeInsets.fromLTRB(0.0, 0.0, 0.0, 20.0),
+                        width: c_width,
+                        child: Text('''Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ver since the 1500s, when an unknown printer took a galley of type''',textAlign: TextAlign.center,style: TextStyle(color: Colors.white,fontSize: 15,),
+                          maxLines: 10,
+                          overflow: TextOverflow.ellipsis,
+                        ),
+                      ),
+                    ],
+                  ),
+                ),
+              ),
+              ButtonBar(
+                alignment: MainAxisAlignment.center,
+                children: <Widget>[
+                  Container(
+                    padding: const EdgeInsets.fromLTRB(0, 0, 60, 0),
+                    child: createButton(MyIcons.done, Colors.green, "done-btn"),
+                  ),
+                  Container(
+                    padding: const EdgeInsets.fromLTRB(60, 0, 0, 0),
+                    child:createButton(MyIcons.clear, Colors.red, "clear-btn"),
+                  ),
+                ],
+              ),
+            ],
+          ),
+        )
```
tout les EdgeInsets, Color et autres éléments de styling devraient etre dans le fichier theme.dart pour qu'on puisse réutiliser plus facilement le code :)

Un-dev commented on netlab/lib/utils/MyIcons.dart:
```diff
@@ -0,0 +1,34 @@
+/// Flutter icons MyFlutterApp
+/// Copyright (C) 2021 by original authors @ fluttericon.com, fontello.com
+/// This font was generated by FlutterIcon.com, which is derived from Fontello.
+///
+/// To use this font, place it in your fonts/ directory and include the
+/// following in your pubspec.yaml
+///
+/// flutter:
+///   fonts:
+///    - family:  MyFlutterApp
+///      fonts:
+///       - asset: fonts/MyFlutterApp.ttf
+///
+///
+/// * Material Design Icons, Copyright (C) Google, Inc
+///         Author:    Google
+///         License:   Apache 2.0 (https://www.apache.org/licenses/LICENSE-2.0)
+///         Homepage:  https://design.google.com/icons/
+///
```
Tu peux tout suppr les commentaires
```suggestion

```

sipe-daniel replied:
c'est ok aussi ici

Un-dev commented on netlab/icons/flutter-icons/my_flutter_app_icons.dart:
```diff
@@ -0,0 +1,33 @@
+/// Flutter icons MyFlutterApp
+/// Copyright (C) 2021 by original authors @ fluttericon.com, fontello.com
+/// This font was generated by FlutterIcon.com, which is derived from Fontello.
+///
+/// To use this font, place it in your fonts/ directory and include the
+/// following in your pubspec.yaml
+///
+/// flutter:
+///   fonts:
+///    - family:  MyFlutterApp
+///      fonts:
+///       - asset: fonts/MyFlutterApp.ttf
+///
+/// 
+/// * Material Design Icons, Copyright (C) Google, Inc
+///         Author:    Google
+///         License:   Apache 2.0 (https://www.apache.org/licenses/LICENSE-2.0)
+///         Homepage:  https://design.google.com/icons/
+///
```
Tu peux suppr les comm ici
```suggestion

```

sipe-daniel replied:
c'est ok ici

Un-dev commented on netlab/lib/utils/customTextField.dart:
```diff
@@ -17,7 +19,7 @@ Widget textfieldWidget(textStyle, codeController, mKeyboardtype) {
+              //borderSide: BorderSide(color: Colors.white),
               borderRadius: BorderRadius.circular(10.0),
             )
         ),
```
dans ce fichier aussi il faudrait mettre les elements de styling dans le [theme.dart ](https://github.com/Un-dev/Netlab/blob/dev-app-codetest/netlab/lib/theme.dart)

sipe-daniel replied:
C'est fait, ici

tchawou-daniel committed:
:recycle:
refactor(my_flutter_app_icons
,home, poll,theme, MyIcons, CustomTextField, theme):

- my_flutter_app_icons and MyIcons: remove comments
- customTexField
- Home : change RaisedButton by CustomButton
- poll, theme and CustomTextField : put the style code inside the theme file

tchawou-daniel committed:
:recycle: refactor(theme.dart): rename the fonction textfieldBorderBSTheme to textfieldBorderOIBSTheme

tchawou-daniel committed:
:recycle: refactor(poll):

tchawou-daniel committed:
:recycle: refactor(theme, poll): restructuration of the project

tchawou-daniel committed:
:recycle:
refactor(theme, poll): restructuration of the project

tchawou-daniel committed:
:recycle: refactor(theme, poll): restructuration of the project

Un-dev committed:
Merge branch 'dev-app' into dev-app-codetest
